tests/_conftest/compiled_account/compiler.py

```diff
@@ -0,0 +1,147 @@
+"""
+DEPRECATED, will be removed soon
```

Comment:
Do we have to remove this? Is there an issue for it?

Reply:
Whole Cairo0 is deprecated, so it just won't be used soon.